### ISSUE TITLE
[Request] Could a toggle be added to disable prevention of placing blocks?

### ISSUE THREAD

**Issue by WesCook** [WhiskyTangoFawks/Expedition#146]
Expedition prevents survival players from placing ore blocks when fracturing is turned on.  This creates a mod conflict with Astral Sorcery, as it requires iron ore be placed on the surface to progress.

Personally, fracturing doesn't concern me.  I don't see it as a viable way of mining blocks.  I'd like to disable this prevention feature if possible.

I wouldn't mind PRing this feature in as a config setting, or forking it if your license permits.  I'd really like to fix this issue for my pack if possible.

Could I get your thoughts, @WhiskyTangoFawks?  Is the Github up to date, so a PR will be viable?

Thank you.

**Comment by WhiskyTangoFawks** [comment]
I'm currently on hiatus from modding while dealing with RL stuff, if/when I come back to active minecraft development I will do this, it's been requested enough, and it's a simple enough feature.

**Comment by WesCook** [comment]
I see.  Thanks for your answer.

Would you permit a private fork to be distributed with a modpack?  I could probably build this feature quickly for my own users, if your Github is updated.

Thank you.

**Comment by WhiskyTangoFawks** [comment]
Yeah, feel free, at the moment I don't know if/when I'll be back.  Just make sure to include a disclaimer that it's a fork.  I don't know how up to date my github is, but you can check the version number.  I don't remember if I updated after seperating the tweaks module out.

**Comment by WesCook** [comment]
Alright, thank you.  I'll look into it.

Best of luck in dealing with the real world.

**Comment by WesCook** [comment]
I think it's after the fork, but because both mods are still in one project it has the features of both included.  I'm not sure if you use a special build script, but this seems to work just fine as-is.

I've gone ahead and added a config to disable ore block placement.  Let me know if you'd be interested in a PR at some point in the future.  Though as you said, it's probably a simple enough thing to add.

Here's the fork.  Please let me know if you have any concerns.  Thanks again for your go-ahead on this.

https://github.com/WesCook/Expedition-Homestead-Edition
